src/Magneto/Magneto.csproj

```diff
@@ -2,8 +2,8 @@
 	<Import Project="..\..\Magneto.Common.props" />
 
 	<PropertyGroup>
-		<TargetFrameworks>netstandard1.1;netstandard1.3;net45;net46</TargetFrameworks>
-		<DisableImplicitFrameworkReferences Condition="'$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'net46'">true</DisableImplicitFrameworkReferences>
+		<TargetFrameworks>netstandard2.0;net46</TargetFrameworks>

```

Comment:
I'm not sure if it's worth the `net46` target. Today I see a lot of people are going full `netstandard2.0`, which will work on 4.6.1 and up.

Reply:
Thanks @slang25. I've now removed the `net46` target too, so we're targeting only `netstandard2.0`.

Reply:
@slang25 Scratch that. Added `net461` back again, according to your own [blog post](https://stuartlang.uk/dotnet-framework-support-for-dotnetnet-standard-2-0/) about it.